CRM/Financeextras/Upgrader.php

```diff
@@ -116,16 +120,4 @@ public function upgrade_1000() {
     return TRUE;
   }
 
-  /**
-   *
-   * This upgrade creates financeextras_exchange_rate table
-   *
-   */
-  public function upgrade_1001() {
-    $this->ctx->log->info('Applying update 1001');
```

Comment:
why did you remove the upgrader ?

Reply:
not needed,for now.

There's  `upgrade_1000()`